GliaWidgets/Lib/Upload/FileUploader.swift

```diff
@@ -104,4 +116,8 @@ class FileUploader {
             state.value = newState
         }
     }
+
+    private func updateLimitReached() {
+        limitReached.value = count == maximumUploads
```

Comment:
Is there a guarantee that there will be no situation when the counter jumps by 2, 3, etc? In that case, it will continue to be false. Maybe a `>=` would be more secure. Or maybe I'm just paranoid and there is no way that would happen 😀

Reply:
Both `addUpload()` methods have a guard to prevent that.

Reply:
I cannot see a way right now of it happening but I would say that the `>=` suggestion makes sense regardless, just in case.